### PR TITLE
fix readme.md of log plugin

### PR DESCRIPTION
**Pull request by mparamonau**
Fixed the readme.md according the PR https://github.com/coredns/coredns/pull/1664.

### CODE REVIEW

File: plugin/log/README.md [hunk 1]
```diff
@@ -28,22 +28,23 @@ log [NAME] [FORMAT]
 * `NAME` is the name to match in order to be logged
 * `FORMAT` is the log format to use (default is Common Log Format)
 
-You can further specify the class of responses that get logged:
+You can further specify the classes of responses that get logged:
 
 ~~~ txt
 log [NAME] [FORMAT] {
-    class [success|denial|error|all]
+    class CLASSES_OF_RESPONSES...
```
Comment (miekg) [inline plugin/log/README.md]: s/CLASSES_OF_RESPONSES/CLASSES

File: plugin/log/README.md [hunk 2]
```diff
@@ -106,3 +107,23 @@ Only log denials for example.org (and below to a file)
...
+    }
+}
+~~~
+
+Log all queries on which we did not get errors
+
+~~~ corefile
+. {
+    log . {
+        class denial success
```
Comment (miekg) [inline plugin/log/README.md:126]: But also:
~~~
class denial
class succes
~~~
?

File: plugin/log/README.md (hunk 1 above)
Comment (mparamonau) [reply plugin/log/README.md]: replaced

File: plugin/log/README.md (hunk 2 above)
Comment (mparamonau) [reply plugin/log/README.md:126]: forgot about it, added it.

**Comment by miekg** [comment]
Thanks!

**Pull request merged**